.github/workflows/tests.yaml

```diff
@@ -305,4 +309,5 @@ jobs:
       - name: Run pytest
         run: |
           . venv/bin/activate
+          pip list | grep 'astroid\|pylint'
```

Comment:
Is that a forgotten debug statement or is this voluntary ?

Reply:
It's a debug statement, but I think a useful one. It can help debug issues when updating astroid and simplify checking if the correct astroid version is used.